librz/core/cautocmpl.c

```diff
@@ -271,6 +271,7 @@ static void autocmplt_cmd_arg_file(RzLineNSCompletionResult *res, const char *s,
 #endif
 		char *tmp = rz_str_newf(fmt, RZ_SYS_DIR, input);
 		if (!tmp) {
+			free(input);
```

Comment:
this isn't necessary, if it's NULL then it's not allocated

Reply:
free (and all the free method in rizin) accepts NULL pointers.